### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by wddcheng**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          python-version: ${{ matrix.python-version }}
+
+      - name: Check
+        shell: bash
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install -e .
+          python3 -m unittest discover -v tests
+          python3 setup.py install
+
```
Comment (dorukkilitcioglu) [inline .github/workflows/ci.yml]: Can you remove some of these blank lines?

File: README.md-e [hunk 2]
```diff
@@ -0,0 +1,149 @@
+[![ci](https://github.com/fidelity/textwiser/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/fidelity/textwiser/actions/workflows/ci.yml) [![PyPI version fury.io](https://badge.fury.io/py/textwiser.svg)](https://pypi.python.org/pypi/textwiser/) [![PyPI license](https://img.shields.io/pypi/l/textwiser.svg)](https://pypi.python.org/pypi/textwiser/) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![Downloads](https://static.pepy.tech/personalized-badge/textwiser?period=total&units=international_system&left_color=grey&right_color=orange&left_text=Downloads)](https://pepy.tech/project/textwiser)
```
Comment (dorukkilitcioglu) [inline README.md-e]: Is this file necessary?

**Comment by dorukkilitcioglu** [comment]
> @wddcheng @dorukkilitcioglu what's the implication of using newer versions of flair? Unit tests fail?

[These CI runs](https://github.com/fidelity/textwiser/runs/5306358526?check_suite_focus=true) are failing, it looks like they changed the flair word embedding API in 0.10. I also find it encouraging that part of the reason why was to make word embeddings fine-tunable in Flair, and their implementation is the same as how I handled it a couple years ago :)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-[![Downloads](https://static.pepy.tech/personalized-badge/textwiser?period=total&units=international_system&left_color=grey&right_color=orange&left_text=Downloads)](https://pepy.tech/project/textwiser)
+[![ci](https://github.com/fidelity/textwiser/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/fidelity/textwiser/actions/workflows/ci.yml) [![PyPI version fury.io](https://badge.fury.io/py/textwiser.svg)](https://pypi.python.org/pypi/textwiser/) [![PyPI license](https://img.shields.io/pypi/l/textwiser.svg)](https://pypi.python.org/pypi/textwiser/) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![Downloads](https://static.pepy.tech/personalized-badge/textwiser?period=total&units=international_system&left_color=grey&right_color=orange&left_text=Downloads)](https://pepy.tech/project/textwiser)
```
Comment (skadio) [inline README.md:1]: nice!

File: requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-flair>=0.6.1
+flair>=0.6.1,<=0.9
```
Comment (skadio) [inline requirements.txt:1]: @dorukkilitcioglu  to confirm my understanding from your reply: If not the upper bound, our unit tests fail with P3.9.

@wddcheng  can you add a comment on this in the changelog please for reference in future.

File: README.md-e (hunk 2 above)
Comment (wddcheng) [reply README.md-e]: removed now

File: .github/workflows/ci.yml (hunk 1 above)
Comment (wddcheng) [reply .github/workflows/ci.yml]: yes!

File: requirements.txt (hunk 4 above)
Comment (wddcheng) [reply requirements.txt:1]: That's right! comments added now

File: CHANGELOG.txt [hunk 5]
```diff
@@ -2,6 +2,12 @@
 TextWiser CHANGELOG
 =====================
 
+-------------------------------------------------------------------------------
+Feb 23, 2022 1.3.1
+-------------------------------------------------------------------------------
+minor:
+- Updated dependency requirements for flair to pass python3.9 unit tests
```
Comment (skadio) [inline CHANGELOG.txt]: minor suggestion, can we be explicit here and mention what the update is? Otherwise, someone has to go back in commits to figure out what's needed for 3.9. Just add sth like the upper bound has to be xxx for flair to pass, sth like that :)

File: requirements.txt (hunk 4 above)
Comment (dorukkilitcioglu) [reply requirements.txt:1]: @skadio correct.

Now that I look at changelog, I do feel like it would be good to have a version bump here. That way, we ensure that installing TextWiser thru PyPI also respects the new requirements.

@wddcheng do you have bandwidth to bump the TextWiser version to 1.3.2 (and all the other bits like updating the docs, uploading to PyPI)? If not, I'm happy to address this.

Comment (wddcheng) [reply requirements.txt:1]: sure! i can do that

Comment (wddcheng) [reply requirements.txt:1]: @dorukkilitcioglu just updated the docs

Comment (dorukkilitcioglu) [reply requirements.txt:1]: Thanks!

**Review by dorukkilitcioglu — APPROVED** [review]
Good to go from my side! Remember to update the PyPI distribution once this is merged.
